bot/main.py

```diff
@@ -172,12 +180,15 @@ async def post_verification(ctx, user):
     try:
         await set_nickname(user, server_config)
     except discord.DiscordException:
-        await ctx.send("Bot should have a role higher than you to change your nickname")
+        await ctx.send(
+            "Bot should have a role higher than you to change your nickname",
+            ephemeral=True,
+        )
 
-    await ctx.send(f"<@{user.id}> has been CAS-verified!")
+    await ctx.send(f"<@{user.id}> has been CAS-verified!", ephemeral=True)
```

Comment:
Just a personal choice/taste, but I'd like this to not be ephemeral. The reasoning here is that I'd like to see the people who successfully verify in the chat

Reply:
ok, understood.
Should this be removed for all commands or just the verify one?

Reply:
I have removed ephemeral attribute from just this line for now.